### DISCUSSION TITLE
Is there a way to delete ALL the artifact files for a repository

### DISCUSSION THREAD

**Discussion by Claudenw** [community/community#30060] [Actions]
I can't seem to find any way to determine the artifact files associated with a repository and delete them.  Is there such?

**Comment by airtower-luna** [comment]
Not in the UI. If you really need to you could script deletion with the API, see https://docs.github.com/en/rest/actions/artifacts.

For *new* artifacts you could [set the retention period for artifacts](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#configuring-the-retention-period-for-github-actions-artifacts-and-logs-in-your-repository) to a really low value, but that doesn't work to clean up old ones.
